pkg/stanza/fileconsumer/internal/fingerprint/fingerprint_test.go

```diff
@@ -22,6 +22,7 @@ func TestNewDoesNotModifyOffset(t *testing.T) {
 	tempDir := t.TempDir()
 	temp, err := os.CreateTemp(tempDir, "")
 	require.NoError(t, err)
+	defer temp.Close()
```

Comment:
Would it make sense to also delete the file?

Reply:
As I understand it, `t.TempDir` will handle deletion of the entire directory. The problem was that deletion failed because the file was still open.